### PR TITLE
Feat/force deploy

### PR DESCRIPTION
**Pull request by VerasThiago**
#### What problem is this solving?

Enable `--force` flag in order to ignore 7 minutes deploy time

#### How should this be manually tested?
On the root  run `yarn link`
On `toolbelt` root run `yarn link "@vtex/cli-plugin-deploy"`
On `toolbelt` root run `vtex-test deploy --force`

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [x] Update `CHANGELOG.md`

### CODE REVIEW

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -12,16 +13,28 @@ export default class Deploy extends CustomCommand {
...
-
   async run() {
-    const {
-      args: { appId },
-      flags: { yes },
-    } = this.parse(Deploy)
-
-    await appsDeploy(appId, { yes })
+    const configClient = ToolbeltConfig.createClient()
+    const { featureFlags } = await configClient.getGlobalConfig()
```
Comment (tergol1) [inline src/commands/deploy.ts:22]: I'm not a specialist in Typescript but I had often see this type of declaration in the beginning of the function. WDYT?

File: src/modules/newDeploy.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    await switchAccount(vendor, {})
+  }
+
+  const context = { account: vendor, workspace: 'master', authToken: session.token }
+  const registry = createRegistryClient(context)
+  const path = `${vendor}.${name}?ignoreWaitPeriod=${force}`
+
+  await registry.validateApp(path, version)
```
Comment (tergol1) [inline src/modules/newDeploy.ts]: **nit** WDYT about putting this in a separate helper?

**Review by tergol1 — APPROVED** [review]
LGTM

File: src/commands/deploy.ts (hunk 1 above)
Comment (tergol1) [reply src/commands/deploy.ts:22]: Never mind, I think it's impossible to do what I was thinking.

File: src/modules/newDeploy.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+const switchToVendorMessage = (vendor: string): string => {
+  return `You are trying to deploy this app in an account that differs from the indicated vendor. Do you want to deploy in account ${chalk.blue(
+    vendor
+  )}?`
+}
+
+const promptDeploy = (app: string) => promptConfirm(`Are you sure you want to deploy app ${app}`)
+
+const deployRelease = async (app: string, force: boolean): Promise<boolean> => {
+  const { vendor, name, version } = parseLocator(app)
```
Comment (tergol1) [inline src/modules/newDeploy.ts:46]: **nit**  would call the variable `name` of `appName`.

Comment (VerasThiago) [reply src/modules/newDeploy.ts:46]: If you call `parseLocator` and get `appName` will return `undefined`

File: src/modules/newDeploy.ts (hunk 2 above)
Comment (VerasThiago) [reply src/modules/newDeploy.ts]: Sure

File: src/modules/newDeploy.ts (hunk 3 above)
Comment (tergol1) [reply src/modules/newDeploy.ts:46]: Huum I think that we would have to change the `parseLocator` (and possibly even other places) so let this that way!

Comment (VerasThiago) [reply src/modules/newDeploy.ts:46]: Ok!

File: src/modules/newDeploy.ts [hunk 4]
```diff
@@ -20,6 +20,14 @@ const switchToVendorMessage = (vendor: string): string => {
 
 const promptDeploy = (app: string) => promptConfirm(`Are you sure you want to deploy app ${app}`)
 
+const execRelease = async (appData: any, token: string, force: boolean) => {
```
Comment (tergol1) [inline src/modules/newDeploy.ts]: I think it would be nice to have an explicit type for `appData`.

**Review by tergol1 — APPROVED** [review]

**Pull request merged**